draft-ietf-quic-transport.md

```diff
@@ -1165,9 +1155,9 @@ discarded if they indicate a different protocol version than that of the
 connection, or if the removal of packet protection is unsuccessful once the
 expected keys are available.
 
-Invalid packets without packet protection, such as Initial, Retry, or Version
-Negotiation, MAY be discarded.  An endpoint MUST generate a connection error if
-it commits changes to state before discovering an error.
+Invalid packets without effective confidentiality protection, such as Initial,
```

Comment:
! The operative protection provided here is data origin authentication, which is why I chose to talk about packet protection instead (rather than introduce people to a new concept).

Reply:
The trouble is that in the -tls draft, these packet types do use the thing we call "packet protection."  Happy to reword here, but I think saying the packets don't have something they clearly do have isn't the right solution.

Reply:
I see your point.  Let's move this to a new issue.